### PR TITLE
Created API for fetching my feedbacks

### PR DESCRIPTION
**Pull request by aish2567**

### CODE REVIEW

File: Server/src/main/java/com/abs/errp/feedback/FeedbackController.java [hunk 1]
```diff
@@ -74,4 +74,12 @@ void removeFeedback(@PathVariable("id") int id) {
 	Optional<Feedback> getFeedback(@PathVariable int id) {
 		return this.feedbackService.getFeedback(id);
 	}
+	
+	@GetMapping("/myFeedbacks")
+	public List<Feedback> getMyFeedbacks(@RequestParam("receiver") int id, @RequestParam("pageNumber") int pageNumber, @RequestParam("pageSize") int pageSize)
+	{
+		ErrpUser e = new ErrpUser();
```
Comment (rbaidABS) [inline Server/src/main/java/com/abs/errp/feedback/FeedbackController.java:81]: Why is the ERRP user object being created here?

File: Server/src/main/java/com/abs/errp/feedback/FeedbackServiceImpl.java [hunk 2]
```diff
@@ -147,5 +150,21 @@ public Optional<Feedback> getFeedback(int id) {
 			throw new ResourceNotFoundException("Feedback not exist with id: " + id);
 		}
 	}
-
+	
+	@Override
+	public List<Feedback> getMyFeedbacks(ErrpUser e,int pageNo, int pageSize) {
```
Comment (rbaidABS) [inline Server/src/main/java/com/abs/errp/feedback/FeedbackServiceImpl.java:155]: Method variables should have meaningful names

**Review by rbaidABS — CHANGES_REQUESTED** [review]

File: Server/src/main/java/com/abs/errp/feedback/FeedbackController.java (hunk 1 above)
Comment (aish2567) [reply Server/src/main/java/com/abs/errp/feedback/FeedbackController.java:81]: Errp user object is created for mapping receiver_id from query parameter with employee id.

File: Server/src/main/java/com/abs/errp/feedback/FeedbackServiceImpl.java (hunk 2 above)
Comment (aish2567) [reply Server/src/main/java/com/abs/errp/feedback/FeedbackServiceImpl.java:155]: Ok mam

**Pull request closed without merging**